src/raw/mod.rs

```diff
@@ -42,18 +40,34 @@ mod requests;
 /// **But, you should not need to worry about all this:** Many functions which accept a
 /// `ColumnFamily` accept an `Into<ColumnFamily>`, which means all of the above types can be passed
 /// directly to those functions.
-#[derive(Default, Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug)]
-pub struct ColumnFamily(String);
+#[derive(Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug)]
+pub enum ColumnFamily {
+    Default,
+    Lock,
+    Write,
+    Version,
+}
 
 impl<T: Into<String>> From<T> for ColumnFamily {
     fn from(i: T) -> ColumnFamily {
-        ColumnFamily(i.into())
+        match i.into().as_str() {
+            "default" => ColumnFamily::Default,
+            "lock" => ColumnFamily::Lock,
+            "write" => ColumnFamily::Write,
+            "versioncf" => ColumnFamily::Version,
+            _ => unreachable!(),
+        }
     }
 }
 
-impl fmt::Display for ColumnFamily {
-    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        self.0.fmt(f)
+impl ToString for ColumnFamily {
```

Comment:
You should always implement Display rather than ToString - ToString is automatically derived for all types which implement Display, but not vice-versa.